### DISCUSSION TITLE
Repository excluded from Hacktoberfest

### DISCUSSION THREAD

**Discussion by Amankr200** [ajay-dhangar/algo#350] [General]
Is this repository excluded from Hacktoberfest?

I want to ensure that contributions made here count towards the Hacktoberfest challenge. Could you please confirm if this project is eligible? @ajay-dhangar 
![Screenshot 2024-10-09 164611](https://github.com/user-attachments/assets/61d68d1e-fc25-4f84-828d-203b20667258)

Thank you!

**Comment by ajay-dhangar** [comment]
Yes, this repository is excluded from Hacktoberfest 2024. However, contributions made to this repository will count towards GSSoC'24 Extd. If you want to contribute to Hacktoberfest 2024, you can work on this repository: https://github.com/CodeHarborHub/codeharborhub.github.io.

![{05144BF3-60AF-4883-B05F-41BEAF94C541}](https://github.com/user-attachments/assets/e011424d-3b0b-4efa-9079-ccb7247e9b3b)
